web/sites/default/config/webform.webform.report_campsite.yml

```diff
@@ -109,16 +109,17 @@ elements: |-
               '!value': ''
         report_blocking_pedestrians:
           '#type': radios
-          '#title': 'Is the campsite blocking pedestrian access?'
-          '#description': 'For example, is it blocking a sidewalk, trail, or path?'
+          '#title': 'Is the campsite blocking pedestrian access, like a sidewalk, trail, or path?'
+          '#description': '<p>The City will also assess all reported sites for potential pedestrian obstructions.&nbsp;</p>'
           '#description_display': before
           '#options': yes_no
           '#options_display': side_by_side
         report_is_ada:
           '#type': radios
           '#title': 'Do you, or someone you legally represent (e.g. child, spouse), have a disability that is affected by this obstruction?'
           '#options':
-            'yes': 'Yes, and I would like to request an accommodation'
+            'yes': 'Yes, and I would like to request an accommodation -- I have, or represent someone with, a disability covered under the Americans with Disabilities Act (ADA). The campsite is creating a barrier that preventing my/their access and I would like to request an accommodation. By checking this option, I acknowledge that the City may contact me to complete the ADA accommodation request process.'
+            'no report': 'Yes, but I do not want to request an ADA accommodation'
```

Comment:
Added this option for users who are reporting a blockage of pedestrian access but don't want to request an accommodation. This change does not change the schema; it just adds another value that may be returned by this field.